### ISSUE TITLE
Black screen shows for a second while switching Day to Night mode and vice versa on Android 9 devices

### ISSUE THREAD

**Issue by pkjvit** [android/plaid#788]
While switching day/night mode black screen shows for a second. The same thing happens when we recreate() the activity.

Could you please help with this or provide any solution?

Thanks,
Pankaj

**Comment by pfmaggi** [comment]
The Activity is recreated when the user switch between light and dark mode. That's why you see the black screen.
Are you suggesting a different behaviour?

**Comment by pkjvit** [comment]
Yes, while recreating activity a black screen shows for a second. but it not coming in previous Android OS versions.

In Android 9+ switch between light to dark mode is not smooth(due to black screen). Can we make it smooth as it works in Android 8 and lower versions?

**Comment by abhi-coding** [comment]
There is also another issue related to this,when you click on change theme many times, we use to get ANR, I am trying to fix it.
